packages/TextArea/__tests__/TextArea.spec.jsx

```diff
@@ -5,9 +5,9 @@ import StandaloneIcon from '@tds/core-standalone-icon'
 import Text from '@tds/core-text'
 import Paragraph from '@tds/core-paragraph'
 import InputFeedback from '@tds/core-input-feedback'
+import { Fade } from '@tds/shared-animation'
```

Comment:
Was this reflected in its package.json?

Reply:
Yes https://github.com/telus/tds-core/blob/4e1224d72a56a3b0a140b62c27fbcd82f0612add/packages/TextArea/package.json#L37